spec/features/idv/steps/in_person/state_id_spec.rb

```diff
@@ -110,7 +110,7 @@
       )
       expect(page).to have_field(
         t('in_person_proofing.form.state_id.identity_doc_address_state'),
-        with: Idp::Constants::MOCK_IDV_APPLICANT[:state_id_jurisdiction],
+        with: Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_JURISDICTION,
```

Comment:
I know you just changed how we were getting this data but rather than getting `state_id_jurisdiction` should it be `MOCK_IDV_APPLICANT_STATE` or `Idp::Constants::MOCK_IDV_APPLICANT[:state]`?

Also, do you think we should have one test (when state jurisdiction is different than state id (doc auth), when address is same as id, we confirm we are using state id (doc auth) and not the state jurisdiction?  I didn't see this test and thought it might be a good one around DAV.

Reply:
Thanks for catching that.  I changed L113 to use `MOCK_IDV_APPLICANT_STATE` for `identity_doc_address_state` and also changed the helper method `fill_out_state_id_form_ok` to use `MOCK_IDV_APPLICANT_STATE` for filling in this field.  I locally ran all the specs that directly called this method and they didn't break after the refactor.  I didn't locally run all the specs that call helper methods which use this method, like `complete_idv_steps_before_address`, but I'll keep an eye on them in the build.

After making this change, the current spec now tests for a scenario where `same_address_as_id` is true and state id jurisdiction differs from identity_doc_address_state.  I think that satisfies your suggestion.